### PR TITLE
Filter tokens not in the liquidity list

### PR DESCRIPTION
**Pull request by anxolin**
Part of this taks https://github.com/gnosis/dex-react/issues/495

I took the simpler approach, I think :) 
We just keep a list of tokens for the liquidity page. The list is maintained there. It looks straightforward and good enough

If in the future we need more fancy stuff we can reevaluate

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr497--dexreact.review.gnosisdev.com

File: src/components/PoolingWidget/index.tsx [hunk 1]
```diff
@@ -182,10 +184,15 @@ const PoolingInterface: React.FC = () => {
...
-  const tokens = useMemo(() => tokenListApi.getTokens(fallBackNetworkId).filter(({ symbol }) => symbol !== 'WETH'), [
-    fallBackNetworkId,
-  ])
+  const tokens = useMemo(() => {
+    return (
+      // Get all the tokens for the current network
+      tokenListApi
+        .getTokens(fallBackNetworkId)
+        // Filter out the tokens not in the list
+        .filter(({ symbol }) => symbol && LIQUIDITY_TOKEN_LIST.includes(symbol))
```
Comment (Velenir) [inline src/components/PoolingWidget/index.tsx]: ~Unnecessary~ small optimization, but `LIQUIDITY_TOKEN_LIST` would be better as a Set

Comment (W3stside) [reply src/components/PoolingWidget/index.tsx]: i agree!

Comment (anxolin) [reply src/components/PoolingWidget/index.tsx]: Done! 14a6190d2dc603d5b421ff22b4e6c6f32128030f

File: src/components/PoolingWidget/index.tsx [hunk 2]
```diff
@@ -30,6 +30,8 @@ import { maxAmountsForSpread, log } from 'utils'
 import { DEFAULT_PRECISION } from 'const'
 import { Link } from 'react-router-dom'
 
+const LIQUIDITY_TOKEN_LIST = new Set(['USDT', 'TUSD', 'USDC', 'PAX', 'GUSD', 'DAI'])
```
Comment (alfetopito) [inline src/components/PoolingWidget/index.tsx]: Shouldn't sUSD be in this list?

File: src/components/PoolingWidget/index.tsx [hunk 3]
```diff
@@ -182,10 +184,15 @@ const PoolingInterface: React.FC = () => {
...
-  const tokens = useMemo(() => tokenListApi.getTokens(fallBackNetworkId).filter(({ symbol }) => symbol !== 'WETH'), [
-    fallBackNetworkId,
-  ])
+  const tokens = useMemo(() => {
+    return (
+      // Get all the tokens for the current network
+      tokenListApi
+        .getTokens(fallBackNetworkId)
+        // Filter out the tokens not in the list
+        .filter(({ symbol }) => symbol && LIQUIDITY_TOKEN_LIST.has(symbol))
```
Comment (alfetopito) [inline src/components/PoolingWidget/index.tsx]:
```suggestion
        .filter(({ symbol }) => LIQUIDITY_TOKEN_LIST.has(symbol))
```

Checking for `undefined` in the set should work, right?

File: src/components/PoolingWidget/index.tsx (hunk 2 above)
Comment (anxolin) [reply src/components/PoolingWidget/index.tsx]: doing in the release

File: src/components/PoolingWidget/index.tsx (hunk 3 above)
Comment (anxolin) [reply src/components/PoolingWidget/index.tsx]: I'm afraid it's expecting a String:
image

Comment (alfetopito) [reply src/components/PoolingWidget/index.tsx]: Ah right, typescript, typescripting :)

**Pull request merged**